pom.xml

```diff
@@ -98,7 +105,8 @@
         <version.awaitility>4.0.1</version.awaitility>
         <version.asm>6.0</version.asm>
         <version.jtwig>5.87.0.RELEASE</version.jtwig>
-        <version.jacoco>0.8.2</version.jacoco>
+        <version.jacoco>0.8.4</version.jacoco>
+        <version.sonar-maven-plugin>3.7.0.1746</version.sonar-maven-plugin>
```

Comment:
Please, replace the dashes with dots to be more consistent with the convention already in use.